docs/source/Support/bskReleaseNotes.rst

```diff
@@ -89,6 +89,7 @@ Version 2.4.0 (August 23, 2024)
 
 - The ``cmake`` command now downloads large Spice data files automatically from the JPL server.
 - Updated Conan/CMake build system to avoid unnecessary recompilations and greatly speed up rebuilds.
+- Added swirl torque information to ref:`THRConfigMsg`, ref:`thrustCMEstimation`, and :ref:`thrusterPlatformState`
```

Comment:
I think a few of the :ref:'s here are just missing prepended colons.

Reply:
Also, as I pushed BSK v2.4.0, please add your release notes to the new beta cycle, not the prior 2.3 beta cycle.

Reply:
Done